services/movie.py

```diff
@@ -0,0 +1,41 @@
+from db.models import Movie
+from django.db.models.query import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,

```

Comment:
It accepts Optional[list[int]] because default already None
Check in all places